Library/Homebrew/github_packages.rb

```diff
@@ -15,7 +15,9 @@ class GitHubPackages
   URL_DOMAIN = "ghcr.io"
   URL_PREFIX = "https://#{URL_DOMAIN}/v2/"
   DOCKER_PREFIX = "docker://#{URL_DOMAIN}/"
-  URL_REGEX = %r{(?:#{Regexp.escape(URL_PREFIX)}|#{Regexp.escape(DOCKER_PREFIX)})([\w-]+)/([\w-]+)}.freeze
+  URL_REGEX = %r{(?:#{Regexp.escape(URL_PREFIX)}|#{Regexp.escape(DOCKER_PREFIX)}|#{Regexp.escape(URL_DOMAIN)}/)
```

Comment:
Why is this needed?

Reply:
To support `brew install ghcr.io/brewsci/bio/seqkit`. The `docker://` prefix is typically omitted when referring to remote OCI tags. It's more ergonomic to type `ghcr.io/brewsci/bio/seqkit` rather than `docker://ghcr.io/brewsci/bio/seqkit`.
See for example the GitHub Packages web interface  installation instructions at https://github.com/orgs/brewsci/packages/container/package/bio/seqkit
> Install from the command line:
> `docker pull ghcr.io/brewsci/bio/seqkit:latest`

Reply:
`brew` is not a Docker command, though. I think the https or docker should be used in this case.